docs/releases/5.1.txt

```diff
@@ -26,6 +26,20 @@ only officially support the latest release of each series.
 What's new in Django 5.1
 ========================
 
+Require authentication for all views by default
```

Comment:
I would prepend "Middleware" otherwise others may panic for a second like I did when I read this, my initial interpretation was that Django would require authentication everywhere by default! :-)
```suggestion
Middleware to require authentication by default
```